### PR TITLE
Chung2014empirical Summary 2

### PR DESCRIPTION
**Pull request by rll2396**
I'm lost as to where the terms with the diagonal matrices come from. I hope my reviewer can help me out here.

### CODE REVIEW

**Comment by TongruiLi** [comment]
Think the diagonal matrix is basically a trainable parameter. In the paper is multiplied with a vector - so basically think of it as a vector that will be element-wise multiplied with another vector.

**Comment by rll2396** [comment]
> 
> 
> Think the diagonal matrix is basically a trainable parameter. In the paper is multiplied with a vector - so basically think of it as a vector that will be element-wise multiplied with another vector.

Why does the GRU not need these terms but the LSTM does?

**Comment by TongruiLi** [comment]
It depends on the exact implementation of LSTM and GRU. To be exact, LSTM can be implemented in a few different implementations - and some do not require the diagonal matrix. If you look at the PyTorch impl for LSTM, the term is replaced by $$b_{ho}$$  . Pretty sure GRU follows the same principle.

**Comment by rll2396** [comment]
> 
> 
> It depends on the exact implementation of LSTM and GRU. To be exact, LSTM can be implemented in a few different implementations - and some do not require the diagonal matrix. If you look at the PyTorch impl for LSTM, the term is replaced by $$b_{ho}$$ . Pretty sure GRU follows the same principle.
> 
> TLDR: the diagonal matrix is not required, it's just the specific impl described in the paper happens to have that.

Okay, so it would be fine to omit the V terms since it's just a summary right?

**Comment by TongruiLi** [comment]
yep

**Comment by rll2396** [comment]
Any other feedback?

**Comment by TongruiLi** [comment]
Besides the review posted above, nope
